modules/installation-azure-config-yaml.adoc

```diff
@@ -0,0 +1,107 @@
+`install-config.yaml` file by using the installation program and modify it.
+====
+
+[source,yaml]
+----
+apiVersion: v1
+baseDomain: example.com <1>
+controlPlane: <2>
+  hyperthreading: Enabled <3> <4>
+  name: master
+  platform:
+    azure:
+      osDisk:
+        diskSizeGB: 512 <5>
+      type: Standard_DS4_v2
+  replicas: 3
+compute: <2>
+- hyperthreading: Enabled <3>
+  name: worker
+  platform:
+    azure:
+      osDisk:
+        diskSizeGB: 512 <5>
+      type: Standard_DS4_v
+  replicas: 5
+metadata:
+  name: test-cluster <1>
+ifdef::without-networking[]
+networking:
+endif::[]
+ifdef::with-networking[]
+networking: <2>
+endif::[]
+  clusterNetwork:
+  - cidr: 10.128.0.0/14
+    hostPrefix: 23
+  machineCIDR: 10.0.0.0/16
+  networkType: OpenShiftSDN
+  serviceNetwork:
+  - 172.30.0.0/16
+platform:
+  azure:
+    region: centralus <1>
+    baseDomainResourceGroupName: resource-group <6>
+pullSecret: '{"auths": ...}' <1>
+sshKey: ssh-ed25519 AAAA... <7>
+----
+<1> Required. The installation program prompts you for this value.
+<2> If you do not provide these parameters and values, the installation program
+provides the default value.
+<3> The `controlPlane` section is a single mapping, but the compute section is a
+sequence of mappings. To meet the requirements of the different data structures,
+the first line of the `compute` section must begin with a hyphen, `-`, and the
+first line of the `controlPlane` section must not. Although both sections
+currently define a single machine pool, it is possible that future versions
+of {product-title} will support defining multiple compute pools during
+installation. Only one control plane pool is used.
+<4> Whether to enable or disable simultaneous multithreading, or
+`hyperthreading`. By default, simultaneous multithreading is enabled
+to increase the performance of your machines' cores. You can disable it by
+setting the parameter value to `Disabled`. If you disable simultanous
+multithreading in some cluster machines, you must disable it in all cluster
+machines.
++
+[IMPORTANT]
+====
+If you disable simultaneous multithreading, ensure that your capacity planning
+accounts for the dramatically decreased machine performance. Use larger
+instance types, such as `m4.2xlarge` or `m5.2xlarge`, for your machines if you
+disable simultaneous multithreading.
+====
+<5> You can specify the size of the disk to use in GB.
+//To configure faster storage for etcd, especially for larger clusters, set the
+//storage type as `io1` and set `iops` to `2000`.
+<6> Specify the name of the resource group that contains the DNS zone for your
+base domain.
+<7> You can optionally provide the `sshKey` value that you use to access the
+machines in your cluster.
++
+[NOTE]
+====
+For production {product-title} clusters on which you want to perform installation
+debugging or disaster recovery, you must provide an SSH key that your `ssh-agent`
+process uses to the installation program.
```

Comment:
Should it be "uses to _access (or authenticate to)_ the installation program"?

Reply:
No, you don't use it to access the installation program. You have to provide it to the installation program so that it can seed your machines with it.

Reply:
I see. It might make more sense to say that "you must provide the installation program with an SSH key that your `ssh-agent` process uses." Or not, if that changes the meaning. Pretty minor, so probably fine like it is.